jekyll/_cci2/url-orbs.adoc

```diff
@@ -0,0 +1,173 @@
+---
+contentTags:
+  platform:
+  - Cloud
+  - Server v4+
+---
+= URL orbs
+:page-description: How to manage URL orbs allow-lists
+:experimental:
+:icons: font
+:page-layout: classic-docs
+
+[#introduction]
+== Introduction
+
+The ability for an organization to use private orbs is no longer dependent on publishing orbs. URL-referenced orbs offer a way to create orbs as well as control access within the organization.
+
+The use of URL orbs is recommended for private orbs where there are additional trust and access controls within the VCS.
+
+
+INFO:
+====
+URL orbs may themselves reference other URL orbs or Registry orbs.
+
+Registry orbs may not reference URL orbs.
+
+Unlike Registry orbs, URL orbs are expanded at the time of use, not the time of publication.
```

Comment:
Maybe something like:
> Because there is no publishing step, URL orbs are expanded when the config is processed at the beginning of a pipeline. Registry orbs are expanded when the orb is published.